pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[tool.poetry]
+name = "medio"
+version = "0.4.1"
+description = "Medical images I/O Python package"
+authors = [
+    "RSIP Vision <info@rsipvision.com>",
+    "Jonathan Daniel <jonathand655@gmail.com>",
+]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/RSIP-Vision/medio"
+keywords = ["medical-images", "IO", "itk", "nibabel", "pydicom"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Intended Audience :: Healthcare Industry",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
Python 3.10 is not supported by ITK yet.
See https://pypi.org/project/itk/#files

Reply:
Right, thanks